packages/ios_platform_images/example/integration_test/ios_platform_images.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'package:ios_platform_images_example/main.dart' as app;
+
+void main() {
```

Comment:
You need to update `script/configs/exclude_integration_ios.yaml`; currently this file isn't actually being run by our tooling. (Thanks for adding it!)